Assets/HoloToolkit/Utilities/Scripts/Billboard.cs

```diff
@@ -53,7 +53,7 @@ private void OnEnable()
         {
             if (TargetTransform == null)
             {
-                TargetTransform = CameraCache.Main.transform;
+                TargetTransform = CameraCache.Main?.transform;
```

Comment:
![image](https://user-images.githubusercontent.com/3580640/42912002-146a6280-8aa2-11e8-9d07-f0b87e741f12.png)

The master project isn't compatible with this syntax.

Reply:
haha! you beat me to it!